packages/flutter/lib/src/foundation/debug.dart

```diff
@@ -97,3 +99,6 @@ String debugFormatDouble(double value) {
   }
   return value.toStringAsFixed(1);
 }
+
+/// A brightness setting that overrides [ui.Brightness] when used through the [BindingBase].
```

Comment:
How do you use it "through the [BindingBase]"?

As implemented, it appears to be a debug setting in the widget layer, determining what brightness the MediaQuery of WidgetApp inserts into the tree, no?

Reply:
I guess it isn't really true - maybe "Used by MediaQuery to override the platform brightness?"

Reply:
Sounds better, we should probably move it to widgets/debug.dart then since its more a debug setting of that layer?

Reply:
I wasn't sure, the window is exposed through BindingBase which is where the Brightness comes from. If one was hypothetically depending only on foundation, it would not be usable if the brightness was in the widget layer

Reply:
Ah, that's fair. If we add some more documentation to this property here I am OK with leaving it where it as.